cli/crates/cli/src/output/report.rs

```diff
@@ -402,8 +405,15 @@ pub(crate) fn check_errors<'a>(
 
     if operation_errors.len() > 0 {
         watercolor::output!("Operation check errors:", @BrightRed);
-        for error in operation_errors {
-            watercolor::output!("- {error}", @BrightRed);
+        for (message, severity) in operation_errors {
+            match severity {
+                SchemaCheckErrorSeverity::Error => {
+                    watercolor::output!("- {message}", @BrightRed);
+                }
+                SchemaCheckErrorSeverity::Warning => {
+                    watercolor::output!("- {message}", @BrightYellow);
+                }
```

Comment:
Should we maybe differentiate these on more than just the colour?  I assume this command would commonly be run on CI where colour support isn't universally available.

Reply:
Good point. I also realize we're not showing warnings when error_count = 0, now. We can/should probably output the warnings separately, before the success / error message.

Reply:
done